Automattic/jetpack
# Add yoast promo in seo settings banner

## Proposed changes:

* Modify the Banner component to support icons without circle design
* Add Yoast promo banner to SEO settings

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Checkout this branch `add/yoast-promo-traffic-block`
* Run your docker env
* Go to `/wp-admin/admin.php?page=jetpack#/traffic`
* See that "Search engine optimization" block promotes Yoast Free:
<img width="1071" alt="CleanShot 2023-03-22 at 18 57 16@2x" src="https://user-images.githubusercontent.com/8419292/227071749-17b6ddf3-4210-46ab-8de4-32a432d4e8ef.png">

* Ensure CTA link is `https://yoa.st/yoast-jetpack-boost?domain=<your_site>`
* Go to "/wp-admin/plugins.php" and install (and activate) Yoast plugin
* Go to `/wp-admin/admin.php?page=jetpack#/traffic`
* See that "Search engine optimization" block promotes Yoast Premium:
<img width="1057" alt="CleanShot 2023-03-22 at 18 58 21@2x" src="https://user-images.githubusercontent.com/8419292/227071831-cea5c18a-9bc9-4c56-96b8-17b66bb2b5df.png">

* Ensure CTA link is `https://yoa.st/yoast-upgrade-jetpack-boost?domain=<your_site>`
* A12s - reach out to me to install Yoast Premium plugin
* Go to `/wp-admin/admin.php?page=jetpack#/traffic`
* The ad should not be displayed anymore

<img width="1057" alt="CleanShot 2023-03-22 at 18 59 54@2x" src="https://user-images.githubusercontent.com/8419292/227072015-1570d0f1-8eba-4adc-92ae-cfc81a527f71.png">
